message_app_server/message_app/send_message.py

```diff
@@ -46,6 +45,27 @@ def messages():
 
     return render_template("messages.html", username=current_user, conversation_id=0, available_conversations=available_conversations)
 
+@send_messages.route("/api/last_message_content", methods=["POST"])
+def api_get_last_message():
```

Comment:
I thought that we can combine all of the endpoints that handle "fetching messages" into one endpoint, with an additional param `options` that specify what needs to be fetched. But we can leave it for later.

Reply:
Yes. Also, this one is actually not just for getting the last message but for getting any message with a specific id.